### ISSUE TITLE
Add README.md

### ISSUE THREAD

**Issue by jmoody** [calabash/Calabash-2.0-x-platform#1]
Hey Tobias,

Can you add a README.md to this repo?

**Comment by TobiasRoikjer** [comment]
@jmoody Yup. I think we'll do this repo all over again when we are about to release Calabash 2.0
